lib/pageflow/file_type.rb

```diff
@@ -26,17 +22,40 @@ class NotFoundError < RuntimeError
     #                          editor_partial: 'pageflow/rainbow/editor/files/file')
     #
     # @param [Hash] options
-    # @option options [ActiveRecord::Base] :model  Required. Model
-    #   representing the file.
+    # @option options [ActiveRecord::Base/String] :model  Required. Model
+    #   representing the file, or name of that model.
     # @option options [String] :editor_partial  Optional. Path of a
     #   partial to include in json representations of the file.
     # @option options [String] :collection_name  Optional. String to
     #   be used in routes. Defaults to `"pageflow_rainbow_file"` for
     #   model `Pageflow::Rainbow::File`.
+    # @option options [Array<FileType>] :nested_file_types
+    #   Optional. Array of FileTypes allowed for nested files. Defaults to [].
     def initialize(options)
-      @model = options.fetch(:model)
+      @model_string_or_reference = options.fetch(:model)
       @editor_partial = options.fetch(:editor_partial, nil)
-      @collection_name = options.fetch(:collection_name, model.model_name.plural)
+      @collection_name_or_blank = options.fetch(:collection_name, '')
```

Comment:
`fetch(..., '')` and the already existing `fetch(..., nil)` above should probably both just be `options[...]`